app/src/main/java/org/mozilla/tv/firefox/settings/SettingsFragment.kt

```diff
@@ -131,19 +132,20 @@ class SettingsFragment : Fragment() {
         val serviceLocator = serviceLocator!!
         val screenController = serviceLocator.screenController
         val fxaRepo = serviceLocator.fxaRepo
+        val telemetryIntegration = TelemetryIntegration.INSTANCE
 
         view.button_firefox_tabs.setOnClickListener {
             // TODO show send tab tutorial
-            // TODO telemetry
+            telemetryIntegration.fxaProfileGetTabsButtonClickEvent()
         }
         view.button_sign_out.setOnClickListener {
             fxaRepo.logout()
-            // TODO telemetry
             screenController.handleBack(fragmentManager!!)
+            telemetryIntegration.fxaProfileSignOutButtonClickEvent()
         }
         view.back_button.setOnClickListener {
-            // TODO telemetry
             screenController.handleBack(fragmentManager!!)
+            telemetryIntegration.fxaProfileGoBackButtonClickEvent()
```

Comment:
Do we need telemetry for hardware back or hardware menu button pressed? Maybe a follow-up?